### PR TITLE
[MERGE] coatils: Add contrib stats

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/18

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/coatils.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        - stats
+        """
+
+        stat_type = match.group(1).strip() if match.group(1) else ''
+        entity = match.group(2).strip() if match.group(2) else ''
+
+        # contrib stats user
+        if stat_type == 'contrib' and entity != '':
+            try:
+                res = list(
```
Comment (Udayan12167) [inline plugins/coatils.py]: Is it not possible to get per user stats rather than getting the whole list?

File: plugins/coatils.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+            yield success
+
+        # bear stats
+        elif stat_type == 'bear' and entity == '':
+            yield 'There are total {} bears.'.format(Coatils.total_bears())
+        # bear stats lang
+        elif stat_type == 'bear' and entity != '':
+            bears = client.list.bears.get().json()
+            all_langs = Coatils.all_langs()
```
Comment (Udayan12167) [inline plugins/coatils.py]: Similarly no endpoint for per language stats.

File: plugins/coatils.py (hunk 1 above)
Comment (meetmangukiya) [inline plugins/coatils.py]: nope, not with current API

File: plugins/coatils.py (hunk 2 above)
Comment (meetmangukiya) [inline plugins/coatils.py]: nope, not in the current API

Comment (meetmangukiya) [inline plugins/coatils.py]: and I don't think there should be a stats endpoint either, we shouldn't create an endpoint for a _application_ IMO

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/126/commits/7a6c0e40a647e8e9f263e4e154883736ee93b291

**Comment by meetmangukiya** [comment]
ack 68665f2

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**